packages/vite-plugin-cloudflare/package.json

```diff
@@ -45,6 +45,7 @@
 		"@cloudflare/unenv-preset": "1.1.1",
 		"@hattip/adapter-node": "^0.0.49",
 		"miniflare": "workspace:*",
+		"tinyglobby": "^0.2.12",
```

Comment:
Is this the same package we use for glob parsing in Wrangler? I feel like we should probably make sure they're the same

Reply:
This is the same package used in Vite (which isn't particularly relevant). Wrangler's glob handling is pretty convoluted. It uses `glob-to-regexp`, which was last published 6 years ago, and then does all the actual matching manually.